kaggle_environments/schemas.json

```diff
@@ -63,7 +63,7 @@
         "default": 1000
       },
       "agentTimeout": {
-        "description": "Maximum runtime (seconds) to initialize an agent.",
+        "description": "Obsolete field kept for backwards compatibility, please use observation.remainingOverageTime.",
```

Comment:
nit: IIUC `actTimeout` is the right field to point to, although `remainingOverageTime` is worth mentioning too.

Reply:
`agentTimeout` represents the initialization timer which is being replaced by `remainingOverageTime` -- users could replace their existing usage of `agentTimeout` with references to `remainingOverageTime` and their code should work as expected.

Reply:
Oops, yeah I didn't read the previous description, sorry.